resources/views/accordion.blade.php

```diff
@@ -57,6 +57,8 @@
                             <span x-show="!openPanel">
                                 @lang('ui::actions.show') {{ $title }}
                             </span>
+                        @elseif ($title instanceof Illuminate\Contracts\Support\Htmlable)
```

Comment:
@ItsANameToo I don't think this would cause issues with existing usages, but we still need to apply classes to that element, so we would still need a way to accept classes for that div. In case of Nodem, we need to apply full width and stuff like that for truncation to work :hmhm: